src/Microsoft.Extensions.OptionsModel/OptionsServiceCollectionExtensions.cs

```diff
@@ -21,6 +20,8 @@ public static IServiceCollection AddOptions(this IServiceCollection services)
             }
 
             services.TryAdd(ServiceDescriptor.Singleton(typeof(IOptions<>), typeof(OptionsManager<>)));
+            // REVIEW: what do we want the lifetime to be?  TODO: make this scoped and try to automatically do IDisposable tracking
```

Comment:
Yes it would reinitialize the options every request, as transient, its worse than that as you get a new TOptions every time.  So I guess that answers the lifetime question, this should be singleton just like the other IOptionsManager then.